BoldGrid/boldgrid-inspirations
# 2.8.0

# 2.8.0 ChangeLog #
* New Feature: Added onboarding checklist to help users get started with Inspirations.
* New Feature: Theme screenshot now displays the specific Crio theme design chosen 
** Resolves #168
* Bug Fix: Image purchase fails with error 'No local cost data'
** Resolves #185

# Testing Instructions #
1. Install Zip from file linked here: [boldgrid-inspirations-2.8.0-rc.1.zip](https://github.com/BoldGrid/boldgrid-inspirations/files/12849229/boldgrid-inspirations-2.8.0-rc.1.zip)

2. If Installing the zip on an existing install, Ensure that the My Inspirations page is replaced with the new onboarding process, and that a percentage is shown for the progress. Otherwise proceed with a new inspiration, or start over the existing one. The "Start Over" button has been relocated to the 'Choose Your Theme' task in the left most column.
3. Proceed through the Inspirations process. You will likely want to repeat this a few times testing traditional Crio themes, as well as themes that use the Custom Page Headers, and with / without a blog. as the cards added will be different
4. Ensure that upon completing an Inspirations theme deployment, that the new Onboarding checklist is show in place of the old My Inspirations page.
5. Ensure that the screenshot shown reflects the theme you chose.
6. Ensure that the cards shown make sense regarding the choices you chose in the deployment process ( Blog, Headers, Sprout, W3TC, etc ).
7. Ensure that all buttons and links work and that they link to the appropriate location.
8. Ensure that progress updates when a task is marked complete / incomplete, and that the progress saves when leaving / returning to the page.
